packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -264,6 +273,11 @@ class _CodeViewState extends State<CodeView>
             child: Expanded(
               child: Scrollbar(
```

Comment:
In addition, if you want this scrollbar to be displayed automatically (without user scrolling operations), you need to set `isAlwaysShown` to true.

Reply:
Thanks! This scrollbar already existed, so I don't think we want to change how it's displayed.